vscode/microsoft-kiota/src/util.ts

```diff
@@ -162,15 +162,24 @@ export function allGenerationLanguagesToString(): string[] {
   return allSupportedLanguages;
 }
 
-export function validateDeepLinkQueryParams(queryParameters: Record<string, string>):
+export interface IntegrationParams {
+  descriptionurl: string;
+  name: string;
+  kind: string;
+  type: string;
+  language: string;
+  source: string;
+};
+
+export function validateDeepLinkQueryParams(queryParameters: Partial<IntegrationParams>):
```

Comment:
Let's have this function return an IntegrationParams object instead of a record<string, string>. This helps create a defined object and also supports autocomplete when working with the parameters downstream